src/test/java/com/artipie/http/HealthSliceTest.java

```diff
@@ -66,7 +64,7 @@ void returnsOkForValidStorage() {
     @Test
     void returnsBadRequestForBrokenStorage() {
         MatcherAssert.assertThat(
-            new HealthSlice(new Settings.Fake(new FileStorage(Paths.get("/")))),
+            new HealthSlice(new Settings.Fake(new FileStorage(null))),
```

Comment:
@genryxy that's a smart fix, but in Artipie project we are trying to avoid using `null` values.

Could you please implement `Storage` interface with all methods throwing exception? It's going to be more verbose, but will make intent more obvious. Using shortcuts is what brought us the problem in the first place.